### ISSUE TITLE
Cant find moves list

### ISSUE THREAD

**Issue by Magnusretrotech** [PanagiotisIatrou/chess-auto-bot#60]
![image](https://github.com/PanagiotisIatrou/chess-auto-bot/assets/96993862/93a1d873-fdc7-4538-a5df-c638b01153de)

I'm using windows.

**Comment by Rafy-exq** [comment]
Yeah Chess.com probably changed their website. So the bot cant find the moves list anymore. Unfortunately I know nothing about HTML so we need to wait until someone more competent fixes this…

**Comment by MmMapIoSpace** [comment]
So, does anyone know what changes on chess.com ? It seems work on bot but not on online match.

**Comment by Rafy-exq** [comment]
Ok so changing line 65 of "chesscom_grabber.py" to:
`            move_list_elem = self.chrome.find_element(By.CSS_SELECTOR, 'wc-move-list.move-list-wrapper-component')`
Solves the issue of not finding the move_list at all. The logic for sorting the moves in away so it can successfully get passed to Stockfish unfortunately doesn’t work anymore. So the bot can’t process the moves. This is something that still needs to get fixed...
Below the HTML_Code for the move_list in a .txt-file.
[Move_List.txt](https://github.com/PanagiotisIatrou/chess-auto-bot/files/15210914/Move_List.txt)

**Comment by Magnusretrotech** [comment]
@PanagiotisIatrou Please fix this

**Comment by Rafy-exq** [comment]
> Ok so changing line 65 of "chesscom_grabber.py" to: ` move_list_elem = self.chrome.find_element(By.CSS_SELECTOR, 'wc-move-list.move-list-wrapper-component')` Solves the issue of not finding the move_list at all. The logic for sorting the moves in away so it can successfully get passed to Stockfish unfortunately doesn’t work anymore. So the bot can’t process the moves. This is something that still needs to get fixed... Below the HTML_Code for the move_list in a .txt-file. [Move_List.txt](https://github.com/PanagiotisIatrou/chess-auto-bot/files/15210914/Move_List.txt)

In addition to that i changed lines 75 and 78 to:
`            moves = move_list_elem.find_elements(By.CSS_SELECTOR, "div.node[data-node] [data-ply]")`
`            moves = move_list_elem.find_elements(By.CSS_SELECTOR, "div.node[data-node] [data-ply]:not([data-processed])")`
And idk how but the bot played one move out of nowhere. Unfortunately it played only one and then stopped working again... I think I reached my limit. I have no idea how to make any progress from here on.
